src/fondant/pipeline/runner.py

```diff
@@ -33,6 +36,39 @@ def run(self, input_spec: str, *args, **kwargs):
 
         subprocess.call(cmd)  # nosec
 
+    def run(
+        self,
+        input: t.Union[Pipeline, str],
+        *,
+        output_path: str = "docker-compose.yml",
```

Comment:
I know super picky, but can we use a common yaml file extension within Fondant. Either `yml` or `yaml`?
btw: accordingly [to the docker compose docs](https://docs.docker.com/compose/compose-file/03-compose-file/#:~:text=The%20default%20path%20for%20a,prefers%20the%20canonical%20compose.yaml%20.) the new preferred default file for docker compose is called `compose.yaml`

Reply:
Fair point ( can we add this to precommit ?). I will make the changes